unit/Makefile

```diff
@@ -96,17 +97,29 @@ CPROVER_LIBS =../src/ansi-c/ansi-c$(LIBEXT) \
 
 OBJ += $(CPROVER_LIBS) testing-utils/testing-utils$(LIBEXT)
 
-TESTS = unit_tests$(EXEEXT) \
-        miniBDD$(EXEEXT) \
+CATCH_TEST = unit_tests$(EXEEXT)
+N_CATCH_TESTS = $(shell \
+                  cat $$(find . -name "*.cpp" \
+                    -and -not -name "miniBDD_new.cpp" \
```

Comment:
:question: How come we exclude this? This seems to be a legitimate catch test?

Reply:
Hmm, maybe we should. The relationship to miniBDD.cpp is a bit weird, though.